src/dotnet-install.sh

```diff
@@ -566,7 +566,8 @@ validate_remote_local_file_sizes()
     local file_size=''
 
     if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-        file_size="$(stat -c '%s' "$downloaded_file")"
+        # hardcode in order to avoid conflicts with GNU stat
```

Comment:
This is the wrong branch - we want to use the BSD stat on macOS, not on Linux, since homebrew on macOS can install the GNU versions.

Reply:
If we have a Mac for testing, you can verify the bad behavior by installing https://formulae.brew.sh/formula/coreutils and running our scripts.

Reply:
fixed.